consensus/ethash/ethash.go

```diff
@@ -412,6 +416,8 @@ type Config struct {
 	PowMode          Mode
 
 	Log log.Logger `toml:"-"`
+	// ECIP-1099
+	EtchashOnBlock *uint64
```

Comment:
With this namespacing `ECIP1099` is equivalent to `Etchash`. I would prefer `ECIP1099Block` since it's more descriptive. This is related to the problematic idea at https://github.com/etclabscore/core-geth/pull/186#discussion_r492692919. 

Since this is the only, and only foreseeable, change to the DAG, I think I'm OK with this pattern, although it isn't as generic-y as my gut would like (deep down I think I want a map of `blockNumber:epochLength`... but probably overkill... for now ;)).